.github/workflows/process_new_pull_request.yml

```diff
@@ -0,0 +1,112 @@
+name: Process New Pull Request
+
+on:
+  pull_request:
+    types: [opened]
+  merge_group:
+    types: [checks_requested]
+  issues:
+    types:
+      - labeled
+  pull_request_target:
+    branches:
+      - develop
+      - release-*
+
+permissions: read-all
+
+jobs:
+  verify_cla:
+    name: Verify CLA
+    runs-on: ubuntu-22.04
+    permissions:
+      issues: write
+      pull-requests: write
+      checks: write
+      statuses: write
+    steps:
+      - name: Checkout repository so that local actions can be used
+        uses: actions/checkout@v4
+      - name: Merge develop branch into the current branch
+        uses: ./.github/actions/merge
+      - name: GitHub Actions from Oppiabot
+        uses: oppia/oppiabot@1.4.0
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+        env:
+          SHEETS_TOKEN: ${{ secrets.SHEETS_TOKEN }}
+          SHEETS_CRED: ${{ secrets.SHEETS_CRED }}
+          SPREADSHEET_ID: ${{ secrets.SPREADSHEET_ID }}
+
+  assign_reviewers:
+    name: Assign Reviewers
+    runs-on: ubuntu-latest
+    needs: verify_cla
+    permissions:
+      issues: write
+      pull-requests: write
+
+    steps:
+      - name: Assign Reviewers
+        uses: actions/github-script@v4
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const { pull_request, repository } = context.payload;
+            const owner = repository.owner.login;
+            const repo = repository.name;
+            const prNumber = pull_request.number;
+            const submitterUsername = pull_request.user.login;
+
+            // Fetch merged PRs of the submitter
+            const mergedPRs = await github.pulls.list({
+              owner,
+              repo,
+              state: "merged",
+              per_page: 5,
+              sort: "created",
+              direction: "desc",
+            });
+
+            const submitterIsNewContributor = mergedPRs.data.length === 0;
+
+            console.log(`User @${submitterUsername} is ${submitterIsNewContributor ? 'a new contributor' : 'not a new contributor'}.`);
```

Comment:
For readability, break after the '(' and indent subsequent lines. Try to keep to within 80 chars.